### ISSUE TITLE
When importing a repo as a project into OpenShft.io, the user is not presented with a full list of all available repos

### ISSUE THREAD

**Issue by ldimaggi** [openshiftio/openshift.io#3927]
Steps to recreate:
* In OpenShift.io, create a new project - specify import
* In a user account with a large number of githib repos, observe that not all the repos are displayed in the drop down presented to the user (in this case, the user account has 37 repos defined - the repo drop downlist only provides access to 20

![screenshot from 2018-07-05 16-32-00](https://user-images.githubusercontent.com/642621/42346662-6a1e27ce-8071-11e8-9dd4-b5b693e29e38.png)

Note that the user is able to search/query for specific repos that are not shown in the repo dropdown:

![screenshot from 2018-07-05 16-32-42](https://user-images.githubusercontent.com/642621/42346806-cc5aa552-8071-11e8-96cb-b10315ef6a14.png)

**Comment by ldimaggi** [comment]
Also note - the repo list shown in the drop down is not presented in alphabetic order.

**Comment by nimishamukherjee** [comment]
Making it Sev3 - as there is a work around

**Comment by vikram-raj** [comment]
@catrobson We need UX input on this.

**Comment by bartoszmajsak** [comment]
@vikram-raj can you share a bit more context on what kind of UX input do we need? 

I would think of investigating if showing partial results of all repositories user has is a bug in the application logic or limitation of the UI component we are using in parallel. Would this approach make sense?

**Comment by catrobson** [comment]
+1 - I need to understand the technical constraints we need to work within in order to give UX/Design guidance on this.

**Comment by vikram-raj** [comment]
@bartoszmajsak @catrobson Currently we are showing only first 20 repositories in the dropdown. So, is it require to show all the repositories? Keep in mind that it is typeahead dropdown and we show the repositories as user type the name of repositories.

**Comment by catrobson** [comment]
Yes, we would want to show all repositories the user has ideally, in alphabetical order as Len states.  Ideally it would be great if we could show the 5 most recent at the top of the dropdown, but this is a nice to have.  The type-ahead would then filter down the list, not search for things not in the list.

Example of a dropdown field like this:
![image](https://user-images.githubusercontent.com/3739227/43841465-43081880-9af1-11e8-8e8d-9a783def3884.png)

Above the line would have most recent, below the line in the dropdown would have a list of all repositories in alphabetical order.

**Comment by vikram-raj** [comment]
@catrobson What will be the max height of the dropdown?

**Comment by vikram-raj** [comment]
@catrobson I added ```400px``` of ```max-height``` to the dropdown. Is it fine?

**Comment by catrobson** [comment]
@SMahil @mindreeper2420  do we have any standards on dropdown height?

**Comment by mindreeper2420** [comment]
@catrobson we do not - dropdowns are governed by PatternFly, which does not have set heights for dropdowns. The only set dimension is a minimum width of `160px`.

**Comment by catrobson** [comment]
Thanks @mindreeper2420 - @vikram-raj I'm good with 400px for this one
